src/Confluent.SchemaRegistry/CachedSchemaRegistryClient.cs

```diff
@@ -128,7 +128,12 @@ private static SubjectNameStrategyDelegate GetValueSubjectNameStrategy(IEnumerab
         /// <param name="config">
         ///     Configuration properties.
         /// </param>
-        public CachedSchemaRegistryClient(IEnumerable<KeyValuePair<string, string>> config)
+        /// <param name="authenticator">
+        ///     The client authenticator
+        ///
+        ///     default: null
+        /// </param>
+        public CachedSchemaRegistryClient(IEnumerable<KeyValuePair<string, string>> config, IHttpAuthenticator authenticator = null)
```

Comment:
this isn't backwards compatible without a recompile, so I think it would be better to get rid of the default value for `authenticator` and make a second constructor that only takes `config` which should call this constructor.